model/registry/http.yaml

```
@@ -118,6 +125,13 @@ groups:
           or a single-item array containing a possibly comma-concatenated string, depending on the way
           the HTTP library provides access to headers.
         examples: ['http.response.header.content-type=["application/json"]', 'http.response.header.my-custom-header=["abc", "def"]']
+      - id: response.size
```

same as previous comment

```suggestion
      - id: response.envelop.size
```